google/cloud/aiplatform/training_jobs.py

```diff
@@ -4582,6 +4678,29 @@ def _add_additional_experiments(self, additional_experiments: List[str]):
         """
         self._additional_experiments.extend(additional_experiments)
 
+    @staticmethod
+    def _create_window_config(
+        column: Optional[str] = None,
+        stride_length: Optional[int] = None,
+        max_count: Optional[int] = None,
+    ) -> Optional[Dict[str, Union[int, str]]]:
+        """Creates a window config from training job arguments."""
+        configs = {
+            "column": column,
+            "stride_length": stride_length,
+            "max_count": max_count,
+        }
+        present_configs = {k: v for k, v in configs.items() if v is not None}
+        if not present_configs:
+            return None
+        if len(present_configs) > 1:
+            raise ValueError(
```

Comment:
Again, we can keep this here if you think there's a really good reason to do validation on the client-side, but we prefer to defer to the service for validation.

Reply:
This is no longer validation of inputs, this is validation that we are passing the input to the API correctly.

Reply:
This is similar to the error the sdk prints when multiple split strategies are specified.